src/contracts/core/StrategyManagerStorage.sol

```diff
@@ -70,8 +70,6 @@ abstract contract StrategyManagerStorage is IStrategyManager {
     // slither-disable-next-line incorrect-shift-in-assembly
     uint256[1] internal _deprecatedStorage;
```

Comment:
what's this? i seem to be forgetting...

Reply:
there's a comment just above:
```
    /* 
     * Reserved space previously used by the deprecated mapping(address => uint256) beaconChainETHSharesToDecrementOnWithdrawal.
     * This mapping tracked beaconChainETH "debt" in case updates were made to shares retroactively.  However, this design was
     * replaced by a simpler design that prevents withdrawals from EigenLayer before withdrawals from the beacon chain, which
     * makes this tracking unnecessary.
     */
```

Reply:
oops thanks!